velox/type/Conversions.h

```diff
@@ -287,6 +287,14 @@ struct Converter<
       return folly::to<T>(v);
     }
   }
+
+  static T cast(const int128_t v) {
+    if constexpr (TRUNCATE) {
```

Comment:
Do we need the TRUNCATE part? We don't take this path in the current implementation.

Reply:
Agree. I put them to make it uniform with existing codes above.

Reply:
It might be better not to include code that's not used. Do we need this 'cast' method? Can we have the caller call `folly::to<T>(v)` directly?

Otherwise, let's add some tests for this method.

Reply:
done.